docs-kits/kits/PURIS Kit/page_adoption-view.md

```diff
@@ -1,232 +1,187 @@
+- I no longer need to circulate Excel spreadsheets and fill them in manually.
+- This means for me less stress and work load.
 
-### Delivery Information
+![Example Use Case](assets/example-use-case.png)
+_Figure 2: Use case for standardized information exchange via PURIS_
 
-Delivery Information includes logistics details and delivery metrics. Logistics details specify the shipment's timing, location, and quantity. Delivery metrics focus on the planned versus actual departure and arrival times, such as shipments leaving a supplier's factory or arriving at a customer's factory. The model provides data on departure and arrival dates and times, locations, quantities shipped, tracking numbers, and other essential logistics information to help coordinate and streamline the delivery process.
+## Semantic Models
 
-**Benefits for LEs, SMEs, and Solution Providers**:
+The information about the provided PURIS standardization objects is detailed in this section. It's used in ongoing business relationships and build-to-order \(BTO\) contexts, not for build-to-stock \(BTS\) situations without existing business ties.
 
-- **LEs**: Enhanced tracking and coordination of incoming and outgoing shipments, enabling better production planning and inventory management.
-- **SMEs**: Improved visibility into delivery timings and quantities helps in adjusting operations, reducing storage costs, and enhancing customer satisfaction.
-- **Solution Providers**: Ability to offer more accurate and timely logistics services, enhancing value to clients through better supply chain transparency and efficiency.
+| Semantic Model                                                                                                                         | Version | Standard name | Description                                                                                                                                                                                                                                                                                           |
+| -------------------------------------------------------------------------------------------------------------------------------------- | :-----: | :-----------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Item Stock](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.item_stock)                                 | v.2.0.0 |    CX-0122    | Item Stock includes materials in the outbound warehouse or inbound at the customer's site. Standardizing its definition and using digital twins for data sharing allows quick inventory updates across the supply chain, improving coordination and addressing supply shortages.                      |
```

Comment:
Here the aspect model and the standard are both v2.0.0

Suggestion: update the link to the static link to the standardized semantic model: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/4d239fc5709f71f39c3cf13581b5bcf960905157/io.catenax.item_stock/2.0.0

Reply:
Link has been adapted